src/azul/service/avro_pfb.py

```diff
@@ -444,7 +444,7 @@ def _entity_schema_recursive(field_types: FieldTypes,
             }
         elif field_type in _nullable_to_pfb_types:
             # Exceptions are fields that do not become lists during aggregation
-            exceptions = ('donor_count', 'total_estimated_cells')
+            exceptions = ('donor_count', 'total_estimated_cells', 'domain', 'value')
```

Comment:
The term "value" is very generic. This could match in other places. Can you catch this one level up, at `accessions`?

Reply:
I should have removed this when I removed the accumulator for non-project entities. It's no longer needed.